samples/net/tftp_client/src/tftp-client.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2023 Kickmaker
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_MODULE_NAME net_tftp_client_app
+#define LOG_LEVEL LOG_LEVEL_DBG
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include <zephyr/drivers/hwinfo.h>
+#include <zephyr/kernel.h>
```

Comment:
Are those includes needed? The sample does not seem to use neither hwinfo nor kernel primitives.